.github/workflows/ci.yml

```diff
@@ -70,7 +70,7 @@ jobs:
           echo "Package version: $PACKAGE_VERSION";
           echo "Specified version: $SPECIFIED_VERSION";
 
-          test "$PACKAGE_VERSION" = "$SPECIFIED_VERSION"
+          test "$PACKAGE_VERSION" = "$SPECIFIED_VERSION" || test "=$PACKAGE_VERSION" = "$SPECIFIED_VERSION"
```

Comment:
Can you update your base PR (the transport redesign) too to make these changes go away from the diff? Merging in master there should help!

Reply:
I think it should be resolved

Reply:
These particular changes should not be visible because they aren't part of any of your PRs, right?

Reply:
There are still lots of changes in this diff that are entirely unrelated :(